static/examples/cstg-secure-signals-example/scripts/scripts.js

```diff
@@ -0,0 +1,84 @@
+const clientSideIdentityOptions = {
+  subscriptionId: 'QRtT141htm',
+  serverPublicKey: 'EUID-X-I-MFkwEwYHKoZIzj0CAQYIKoZIzj0DAQcDQgAEYla1YQ6N+surF4Yjaax46mPxCv7ixCR0zd3AYV5D8nhEVbQOuhs+GGxp0rUVpucJGQxNvkINwdSeCTpyzLMtFA==',
+};
+
+function updateGuiElements(state) {
+    $('#targeted_advertising_ready').text(
+        __euid.getAdvertisingToken() ? 'yes' : 'no'
+    );
+    const token = __euid.getAdvertisingToken();
+    $('#advertising_token').text(String(token));
+    $('#login_required').text(
+        __euid.isLoginRequired() || __euid.isLoginRequired() === undefined ? 'yes' : 'no'
+    );
+    $('#identity_state').text(String(JSON.stringify(state, null, 2)));
+
+    const euidLoginRequired = __euid.isLoginRequired();
+    if (euidLoginRequired) {
+        $('#login_form').show();
+        $('#logout_form').hide();
+    } else {
+        $('#login_form').hide();
+        $('#logout_form').show();
+    }
+
+    const secureSignalsStorage = localStorage['_GESPSK-euid.eu'];
+    if (token && !secureSignalsStorage) {
+        // token is valid but secure signals has not been refreshed.  reload the page
```

Comment:
caps for Secure Signals, period at the end.

// Token is valid but Secure Signals has not been refreshed. Reload the page.